scripts/src/middleware.ts

```diff
@@ -16,3 +16,31 @@ export function logRequest(req: express.Request, res, next) {
 	logger.info(`start handling request: ${ req.method } ${ req.path } with ${ req.rawHeaders }`);
 	next();
 }
+
+export function notFoundHandler(req: express.Request, res: express.Response, next: express.NextFunction) {
+	// log.error(`Error 404 on ${req.url}.`);
+	res.status(404).send({ status: 404, error: "Not found" });
+}
+
+export type ApiError = {
+	status: number;
+	error: string;
+};
+
+export function generalErrorHandler(err: any, req: express.Request, res: express.Response, next: express.NextFunction) {
+	let message = "Error";
+
+	message += `\n\tmethod: ${ req.method }`;
+	message += `\n\tpath: ${ req.url }`;
+	message += `\n\tpayload: ${ JSON.stringify(req.body) }`;
```

Comment:
Template stings are multiline, if you are using HTML try to use the <pre> tag or the <br> tag instead of this horrid string build you got going on here.